dspace-api/src/main/java/org/dspace/qaevent/service/dto/OpenaireMessageDTO.java

```diff
@@ -0,0 +1,173 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.qaevent.service.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Implementation of {@link QAMessageDTO} that model message coming from OPENAIRE.
+ * @see <a href="https://graph.openaire.eu/docs/category/entities" target="_blank"> see </a>
+ * @author Luca Giamminonni (luca.giamminonni at 4science.it)
+ *
+ */
+public class OpenaireMessageDTO implements QAMessageDTO {
+
+    @JsonProperty("pids[0].value")
+    private String value;
+
+    @JsonProperty("pids[0].type")
+    private String type;
+
+    @JsonProperty("instances[0].hostedby")
+    private String instanceHostedBy;
```

Comment:
can you add java docs for these properties?

Reply:
On top of the class declaration the url to the official OpenAIRE documentation has been linked already.